audio-guestbook.ino

```diff
@@ -38,6 +44,10 @@
 #define HOOK_PIN 0
 #define PLAYBACK_BUTTON_PIN 1
 
+// comment his out, if your handheld OPENS the contact on lift

```

Comment:
Found a typo here :)

Reply:
Thanks!
Corrected and added a lot more configuration options and made the sketch much easier to use for the beginner.